app/start.js

```diff
@@ -0,0 +1 @@
+require('./bin/www');
```

Comment:
In this way, when combine with nodemon, we don't need to copy /bin/www initialization code to app.js. This is the best solution I can find so far.https://github.com/remy/nodemon/issues/330#issuecomment-44370399